frontend/src/component/common/SegmentItem/SegmentItem.tsx

```diff
@@ -32,16 +39,14 @@ const StyledAccordion = styled(Accordion)(({ theme }) => ({
 }));
 
 const StyledAccordionSummary = styled(AccordionSummary)(({ theme }) => ({
-    padding: 0,
+    padding: theme.spacing(0, 3),
     fontSize: theme.typography.body2.fontSize,
     minHeight: 'unset',
-    '.MuiAccordionSummary-content, .MuiAccordionSummary-content.Mui-expanded': {
-        margin: 0,
-    },
 }));
 
 const StyledAccordionDetails = styled(AccordionDetails)(({ theme }) => ({
-    padding: theme.spacing(2, 0, 1),
+    borderTop: `1px dashed ${theme.palette.divider}`,
```

Comment:
Ooh, a dashed border? That's ... dashing? I don't think I've seen that in sketches before? Is it a new style we're trying out?

Reply:
We have it in context. I think it looks neat